src/ontology/edits/oeo-model.omn

```diff
@@ -470,1114 +470,1424 @@ Class: <http://purl.obolibrary.org/obo/IAO_0000300>
+        OEO_00000270
     
     
-Individual: Econometric
+Individual: OEO_00000163
 
+    Annotations: 
+        rdfs:label "false"
+    
     Types: 
-        MethodicalFocus
+        OEO_00000078
     
     
-Individual: EconomicEquilibrium
+Individual: OEO_00000170
 
+    Annotations: 
+        rdfs:label "fortran"
+    
     Types: 
-        MethodicalFocus
+        OEO_00000382
     
     
-Individual: False
+Individual: OEO_00000179
 
+    Annotations: 
+        rdfs:label "fuzzy logic"
+    
     Types: 
-        BooleanVariable
+        OEO_00000265
     
     
-Individual: Fortran
+Individual: OEO_00000180
 
+    Annotations: 
+        rdfs:label "gams"
+    
     Types: 
-        SoftwareFramework
+        OEO_00000382
     
     
-Individual: FuzzyLogic
+Individual: OEO_00000187
 
+    Annotations: 
+        rdfs:label "gdx"
+    
     Types: 
-        MathematicalConcept
+        OEO_00000120
     
     
-Individual: GAMS
+Individual: OEO_00000195
 
+    Annotations: 
+        rdfs:label "glpk"
+    
     Types: 
-        SoftwareFramework
+        OEO_00000392
     
     
-Individual: GLPK
+Individual: OEO_00000196
 
+    Annotations: 
+        rdfs:label "gnu"
+    
     Types: 
-        Solver
+        OEO_00000382
     
     
-Individual: GNU
+Individual: OEO_00000203
 
+    Annotations: 
+        rdfs:label "gurobi"
+    
     Types: 
-        SoftwareFramework
+        OEO_00000392
     
     
-Individual: Gurobi
+Individual: OEO_00000215
 
+    Annotations: 
+        rdfs:label "hybrid"
+    
     Types: 
-        Solver
+        OEO_00000057
     
     
-Individual: Hybrid
+Individual: OEO_00000225
 
+    Annotations: 
+        rdfs:label "inactive"
+    
     Types: 
-        AnalyticalApproach
+        OEO_00000281
     
     
-Individual: Inactive
+Individual: OEO_00000244
 
+    Annotations: 
+        rdfs:label "java"
+    
     Types: 
-        Modus
+        OEO_00000382
     
     
-Individual: Java
+Individual: OEO_00000254
 
+    Annotations: 
+        rdfs:label "linear programming"
+    
     Types: 
-        SoftwareFramework
+        OEO_00000265
     
     
-Individual: LinearProgramming
+Individual: OEO_00000262
 
+    Annotations: 
+        rdfs:label "macro- economic"
+    
     Types: 
-        MathematicalConcept
+        OEO_00000270
     
     
-Individual: MOSEK
+Individual: OEO_00000266
 
+    Annotations: 
+        rdfs:label "math prog"
+    
     Types: 
-        Solver
+        OEO_00000382
     
     
-Individual: MS_Excel
+Individual: OEO_00000267
 
+    Annotations: 
+        rdfs:label "matlab"
+    
     Types: 
-        SoftwareFramework
+        OEO_00000382
     
     
-Individual: Macro-Economic
+Individual: OEO_00000273
 
+    Annotations: 
+        rdfs:label "mixed integer linear programming"
+    
     Types: 
-        MethodicalFocus
+        OEO_00000265
     
     
-Individual: MathProg
+Individual: OEO_00000280
 
+    Annotations: 
+        rdfs:label "modellca"
```

Comment:
Not part of this issue/pull request. But I think it should be _**M**odel**i**ca_ (see https://en.wikipedia.org/wiki/Modelica) and we could fix it now.

```suggestion
        rdfs:label "modelica"
```